.yarnrc.yml

```diff
@@ -1,5 +1,7 @@
 nodeLinker: node-modules
 
+checksumBehavior: update
```

Comment:
Been seeing some issues resolving merge conflicts with checksums. Turning this on fixes the problem so I figured I'd just leave it on.